Modelica/Mechanics/MultiBody/Examples/Elementary/UserDefinedGravityField.mo

```diff
@@ -52,7 +52,7 @@ This example demonstrates a user defined gravity field.
 Function \"world.gravityAcceleration\" is redeclared to function
 <a href=\"modelica://Modelica.Mechanics.MultiBody.Examples.Elementary.Utilities.theoreticalNormalGravityWGS84\">theoreticalNormalGravityWGS84</a>
 that computes the theoretical gravity of the
-<a href=\"https://earth-info.nga.mil/coordsys/coord-download.php?file=website/NGA.STND.0036_1.0.0_WGS84.pdf\">WGS84 ellipsoid earth model</a> at and close to
+<a href=\"https://earth-info.nga.mil/php/download.php?file=coord-wgs84\">WGS84 ellipsoid earth model</a> at and close to
```

Comment:
I would prefer if we found a more stable reference for this; but it doesn't seem like it exists